app/views/follow_requests/create.js.erb

```diff
@@ -0,0 +1,8 @@
+console.log("follow request sent")
```

Comment:
Nitpick. Can you remove the `console.log` from your code, it makes your code look cleaner.